plugins/modules/ise_radius_integration_workflow_manager.py

```diff
@@ -1217,6 +1218,27 @@ def update_auth_policy_server(self, ipAddress):
             if is_ise_server:
                 trusted_server = self.want.get("trusted_server")
                 self.accept_cisco_ise_server_certificate(ipAddress, trusted_server)
+                response = self.dnac._exec(
+                    family="system_settings",
+                    function='get_authentication_and_policy_servers',
+                    params={"is_ise_enabled": True}
+                )
+                response = response.get("response")
+                if response is None:
+                    self.msg = "Failed to retrieve the information from the API 'get_authentication_and_policy_servers' of {0}." \
+                               .format(ipAddress)
+                    self.status = "failed"
+                    return
+
+                ise_server_details = get_dict_result(response, "ipAddress", ipAddress)
+                ise_state_list = ["FAILED", "INPROGRESS"]
```

Comment:
For discussion: 

Can we use set instead of list? 

https://stackoverflow.com/questions/8929284/what-makes-sets-faster-than-lists#:~:text=Generally%20the%20lists%20are%20faster,average%20is%20O(1).

Generally the lists are faster than sets. But in the case of searching for an element in a collection, sets are faster because sets have been implemented using hash tables. So basically Python does not have to search the full set, which means that the time complexity in average is O(1).

Your inputs are welcome